src/core_plugins/kbn_vislib_vis_types/public/editors/pie.html

```diff
@@ -2,14 +2,17 @@
   <div class="form-group">
     <div class="kuiSideBarSectionTitle">
       <div class="kuiSideBarSectionTitle__text">
-        Pie Settings
+        <span i18n-id="kbnVislibVisTypes.editors.pie.pieSettings" i18n-default-message="Pie Settings"></span>
```

Comment:
do not create extra `span` tag, use `<div class="kuiSideBarSectionTitle__text">` instead